api/src/main/java/org/pmiops/workbench/leonardo/LeonardoApiClientImpl.java

```diff
@@ -552,12 +560,14 @@ public void createApp(CreateAppRequest createAppRequest, DbWorkspace dbWorkspace
           workbenchConfigProvider.get().firecloud.userApps.rStudioDescriptorPath);
     }
 
+    // For CROMWELL save the generatedUserAppName in DB
+    String userAppName =
+        appType.equals(AppType.CROMWELL)
+            ? getCromwellAppName(dbWorkspace)
```

Comment:
I don't like "get" as a name here because sometimes it creates.  something like getOrGenerate maybe?